app/AccountancyModule/PaymentModule/components/PaymentDialog.php

```diff
@@ -174,13 +180,21 @@ private function updatePayment(ArrayHash $values) : void
         $this->flashMessage('Platba byla upravena', 'success');
     }
 
+    /**
+     * @return string[]
+     */
+    private function processEmails(?string $emails) : array
+    {
+        return $emails === null ? [] : explode(MyValidators::EMAIL_SEPARATOR, $emails);
```

Comment:
Tady chybí ořezání whitespaců (jako je to prováděno ve validátoru).

Např. `foo1@bar.cz, foo2@bar.cz` pro jde validací, ale do plateb pak půjde `["foo1@bar.cz", " foo2@bar.cz"]`

Reply:
jasně, upraveno